Maestro/subscriptions.json

```diff
@@ -717,6 +717,36 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/3.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/3.0/packages.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/3.0.1/packages.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/3.0.0/Latest.txt",
```

Comment:
The prodcon v1 ones can actually be removed, since they will never happen for 3.0. I think dotnet/standard is upgraded through Maestro++ now, too.

Reply:
What are you referring to w/ "The prodcon v1 ones" ?

Reply:
The ones with `product` in them are semaphores for prodcon v1 builds, which contain full stack build manifests. They aren't actually dependencies on CLI, the prodcon v1 builds are just building the CLI product.